rpc/tests/utils/client.rs

```diff
@@ -45,7 +45,7 @@ pub async fn new_test_client(auth_level: AuthLevel) -> Result<Client> {
     let client = Client::new(&url, token.as_deref()).await?;
 
     // minimum 2 blocks
-    client.header_wait_for_height(2).await?;
```

Comment:
in this block, the account we use for blob submission should be founded with tia, why this change?

Reply:
when testing against mocha, if I had my node sync from a recent height and trusted hash, waiting for height 2 would trigger a warning on the node `header not found` (celestia-node), so removing it allowed me to test this, but we could have an env for this height and that way the test does not assume you have a node synced from genesis etc

Reply:
We can instead do a loop here, where we request network head and break if it is >2 and sleep otherwise

Reply:
revisiting here, I think the problem with that (from my own testing), is that if I sync a node from a trusted hash and you request to wait for the block height `2`, the node returns "header not found". Maybe having an env var to control which height to wait for gives more flexibility?